vaktija-desklet@MuxBH28/README.md

```diff
@@ -0,0 +1,51 @@
+# Vaktija Desklet
+
+## Table of Contents
+- [Overview](#overview)
+  - [How It Works](#how-it-works)
+  - [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Configuration](#configuration)
+- [Links](#links)
+- [Author](#author)
+
+## Overview
+
+Vaktija Desklet is a simple desklet that displays prayer times using data from the vaktija.ba API.
+
+### How It Works
+
+- The desklet displays current prayer times for a given location.
+- Data is fetched from the vaktija.ba API and updated in real-time.
+
+### Features
+
+- Displays prayer times for the selected location.
+- Real-time data updates.
+
+## Installation
+
+1. Download the files from the [GitHub repository](https://github.com/MuxBH28/vaktija-desklet).
+2. Copy the downloaded files to `~/.local/share/cinnamon/desklets/vaktija-desklet@muxbh28/`.
+3. Restart Cinnamon.
```

Comment:
Users will be able to install locally once this is merged.

Reply:
Fixed it. Just waiting for merge now

Reply:
Do you need a hand with translation support?